src/acl.c

```diff
@@ -91,6 +91,8 @@ struct ACLUserFlag {
 } ACLUserFlags[] = {
     {"on", USER_FLAG_ENABLED},
     {"off", USER_FLAG_DISABLED},
+    {"skip-sanitize-payload", USER_FLAG_SANITIZE_PAYLOAD_SKIP},
+    {"sanitize-payload", USER_FLAG_SANITIZE_PAYLOAD},
```

Comment:
It may be a trivial comment, but the only importance of the order of these flags is in ACLDescribeUser, which would make this come first. I'd rather have it towards the end :)